AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/AWSModelReconciliationQueue.swift

```diff
@@ -133,11 +133,19 @@ final class AWSModelReconciliationQueue: ModelReconciliationQueue {
     func enqueue(_ remoteModel: MutationSync<AnyModel>) {
         let reconcileOp = ReconcileAndLocalSaveOperation(remoteModel: remoteModel,
                                                          storageAdapter: storageAdapter)
-        reconcileAndLocalSaveOperationSink = reconcileOp.publisher.sink(receiveCompletion: { error in
-            self.modelReconciliationQueueSubject.send(completion: error)
+        var reconcileAndLocalSaveOperationSink: AnyCancellable?
+
+        reconcileAndLocalSaveOperationSink = reconcileOp.publisher.sink(receiveCompletion: { completion in
+            switch completion {
+            case .finished:
+                self.reconcileAndLocalSaveOperationSinks.remove(reconcileAndLocalSaveOperationSink)
```

Comment:
We should remove the sink on any completion, not just a `.finished`

Reply:
thank you!